python/private/common/providers.bzl

```diff
@@ -136,6 +138,11 @@ the same conventions as the standard CPython interpreter.
             "filesystem path to the interpreter on the target platform. " +
             "Otherwise, this is `None`."
         ),
+        "interpreter_version_info": (
```

Comment:
Let the bikeshedding on the name commence...

Reply:
I'm OK with `interpreter_version_info`, as that is what it's representing. `python_version_info` or `version_info` would also be OK IMHO.